code/index.html

```diff
@@ -2,17 +2,93 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
+    <title>Giannina Portfolio</title>
     <link rel="stylesheet" href="./style.css" />
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Amatic+SC:wght@400;700&family=Caveat:wght@400;500;600;700&family=Neucha&family=Permanent+Marker&family=Rock+Salt&family=Shadows+Into+Light&family=Yanone+Kaffeesatz:wght@200;300;400;500;600;700&family=Zen+Kurenaido&display=swap');
+      </style> 
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+      <img src="./Logo_02.png" alt="Illustrated Icon">
+      <nav>
+        <a href="#"> Home</a>
+        <a href="#">About me</a>
+        <a href="#">Contact</a>
+      </nav>
     </header>
 
-    <main class="">
-      <!-- The main container to use as the flex parent for your card layout, the main content of the site should be in here.  -->
-    </main>
+    <section class="Introduction-Work">
+      <section class="mini-menu-links">
+        <a href="./Illustrations.html">Illustration</a>
+        <div></div>
+        <a href="#">Paintings</a>
+        <div></div>
+        <a href="#">Motion Design</a>
+      </section>
+      <img src="./Backseat_Still.png" alt="Still of Backseat Movie">
+      <div>
+        <h1> Animation</h1>
+        <h2> On this page you'll find all the Animation projects
+          i've worked on so far.
+        </h2>
+      </div>
+    </section>
+
+    <section class="Animations">
+      <section class="Backseat-Card">
+        <section class="Animations-Titel">
+          <img src="./BACKSEAT_Poster.png" alt="Backseat Icon">
+          <div>
+            <h3> 2021 Backseat</h3>
+            <h4> Graduation Movie</h4>
+          </div>
+        </section>
+        <section class="Backseat-trailer-link">
+          <a href="https://vimeo.com/563644712" target="_blank" rel=“noopener noreferrer“> <img src="./Final_Graded_v03.png" alt=""></a>
```

Comment:
I LOVE your videos. They look really nice! A little tip when it comes to Vimeo: you can find an embed-code snippet on the Vimeo site, to be able to embed a player and let the visitor see the movie at your website instead of being transferred to Vimeo. 👍🏻